Tests/Indicators/MovingAverageTypeExtensionsTests.cs

```diff
@@ -33,6 +33,21 @@ public void CreatesCorrectAveragingIndicator()
             indicator = MovingAverageType.Wilders.AsIndicator(1);
             Assert.IsInstanceOf(typeof(ExponentialMovingAverage), indicator);
 
+            indicator = MovingAverageType.LinearWeightedMovingAverage.AsIndicator(1);
+            Assert.IsInstanceOf(typeof(LinearWeightedMovingAverage), indicator);
+
+            indicator = MovingAverageType.DoubleExponential.AsIndicator(1);
+            Assert.IsInstanceOf(typeof(DoubleExponentialMovingAverage), indicator);
+
+            indicator = MovingAverageType.TripleExponential.AsIndicator(1);
+            Assert.IsInstanceOf(typeof(TripleExponentialMovingAverage), indicator);
+
+            indicator = MovingAverageType.Triangular.AsIndicator(1);
+            Assert.IsInstanceOf(typeof(TriangularMovingAverage), indicator);
+
```

Comment:
Maybe we can automate this test by decorating the enum members with an attribute that points to the type they create, then this method could be written using reflection... not for now, but thoughts for later as we add to this enumeration